### ISSUE TITLE
Add `sits` and `rstac`

### ISSUE THREAD

**Issue by kadyb** [cran-task-views/Spatial#49]
I think they should be also included in the [SpatioTemporal](https://github.com/cran-task-views/SpatioTemporal).

**Comment by rsbivand** [comment]
PRs please!

**Comment by kadyb** [comment]
CC: @rolfsimoes, @gilbertocamara

**Comment by kadyb** [comment]
I proposed this: #50, but we can close it and use your proposal. What do you think?

**Comment by gilbertocamara** [comment]
Dear @kadyb, thanks for your response. Let's then use my proposal, if we may.

**Comment by rsbivand** [comment]
Yes, but you can substitute these texts for yours and commit, push to your PR.

**Comment by kadyb** [comment]
@gilbertocamara, would it be fine for you if I update current PR with your descriptions?

**Comment by rsbivand** [comment]
Of course with the correct link markup, as in the original PR.
